packages/ai-core/__tests__/AiSlice.modelSelection.test.ts

```
@@ -264,6 +295,43 @@ describe('AiSlice model selection', () => {
     expect(sendMessage).toHaveBeenCalledWith({text: 'hello'});
   });
 
+  it('clears draft context after capturing it for a run', async () => {
+    let store!: ReturnType<typeof createTestStore>;
+    store = createTestStore({
```

ESLint

[error] 'store' is never reassigned. Use 'const' instead.

(prefer-const)